crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -206,7 +206,10 @@ impl ConductorConfig {
 
     /// Check if the config is set to use a rendezvous bootstrap server
     pub fn has_rendezvous_bootstrap(&self) -> bool {
-        self.network.bootstrap_service == Some(url2::url2!("rendezvous:"))
+        match url2::try_url2!("rendezvous:") {
```

Comment:
Can this panic? It's a known string that is a valid URL prefix

Not necessarily a bad thing to remove the panic but this does seem like an error you'd want to catch rather than just returning a boolean here if somebody made a bad edit

Reply:
It can't panic really, but just another infallible url2 usage I spotted, I can keep it since we are sure this won't panic. Reverted